docs/instructions.md

```diff
@@ -75,10 +74,44 @@ After following the steps in [Quick Start](#quick-start-app) above:
 
 ### App Settings ###
 
+#### iOS
+
+To change your general settings for the Whisper app, click the Settings button on the start screen. You will be automatically redirected to the Whisper section of your device's Settings app, which contains two lists of options. The first, "Allow Whisper to Access," lets you control how the app interacts with other features on your device. The second, "Whisper Settings," lets you control how the app itself behaves. 
+
+**Allow Whisper to Access** has four sections:
+
+* **Bluetooth** — allowing Whisper access to Bluetooth is what lets you start and join in-person conversations automatically when you tap Whisper or Listen. We highly recommend allowing Bluetooth access!
+* **Siri & Search** — tapping this option brings you to a menu where you can refine how much access Apple has to how you use Whisper, as well as whether you can use the built-in voice command functions to find and open Whisper
+* **Notifications** — tapping this option brings you to a menu where you can change what notifications (if any) the Whisper app will give you, the style of those notifications, and whether you'll see whispers on your lock screen
+  * This is where you choose whether the app alerts you with a sound when a new whisper comes in; to choose a particular alert sound, go to "Alert Sound" in Whisper Settings
+* **Background App Refresh** — this option is permanently set to on [explanation from Dad]
+
+ **Whisper Settings** has five sections:
+
+* **Remember name from last session** — turning this option on means you don't have to type in your name every time you join a conversation in the app. If multiple people use Whisper on the same device, it may make more sense to turn this option off so no one accidentally joins a conversation under the wrong name.
+* **Require Bluetooth listeners to pair** — [[I don't know what this means]]
+* **Read whisper aloud** — turning this option on means your device will read incoming whispers aloud — [[DAD IS THIS OPTION NECESSARY NOW?]]
```

Comment:
Yes, absolutely, for when people can't look at their device (e.g., Lisa is doing the dishes in the kitchen and Bill is far enough away that she can't hear his device).  Or when I was on the ladder in the kitchen and Bill was in the garage and we were doing connectivity checks on wires and circuit breakers.

Reply:
I understand what the purpose is, but I think we should take it out and let that be decided in-app. See issue #26

Reply:
Please prioritize #26 because then I can just remove this line